dimension_reduction/pca_example.Rmd

```diff
@@ -0,0 +1,117 @@
+---
+title: "PCA"
+author: "ALSF CCDL - Candace Savonen"
+output:   
+  html_notebook: 
+    toc: true
+    toc_float: true
+---
+
+*Purpose*: This notebook illustrates how to perform Principal Components 
+Analysis and plot the scores using ggplot2. 
+
+Recommended articles on PCA: 
+- [Overall Explanation by Matt Brems](https://towardsdatascience.com/a-one-stop-shop-for-principal-component-analysis-5582fb7e0a9c)
+- [A visual explanation of PCA](http://setosa.io/ev/principal-component-analysis/)
+
+## 1) Set up libraries
+
+```{r}
+# Magrittr pipe
+`%>%` <- dplyr::`%>%`
+
+# Load ggplot2 for graphing
+library(ggplot2)
+
+# Make a results directory if it isn't created yet
+if (!dir.exists("results")) {
+  dir.create("results")
+}
+```
+
+## 2) Import and set up data
+Data downloaded from refine.bio are in tab separated values ("tsv") files and 
+include a metadata in a separate tsv file. 
+These data set was obtained from [refine.bio](https://www.refine.bio/). 
+
+```{r Import refine.bio data}
+# Put name of data file
+data.filename <- file.path("data", "GSE71270.tsv")
+
+# Read in data tsv file
+df <- readr::read_tsv(data.filename, progress = FALSE) %>% 
+  tibble::column_to_rownames('X1') 
+# Make the gene column the rownames so the gene names are out of the way for calculations
```

Comment:
I would move this comment to the line above `tibble::column_to_rownames` because in the HTML file it's rendered as a code chunk that only has this comment.